app.rb

```diff
@@ -4,170 +4,12 @@
 require_relative './book'
 
 class App
+  include ShowMenu
```

Comment:
- [ ] KIndly make sure to import the `show_menu.rb` file in the main app to make the app includes it .so that the app can function properly. currently, your app is crashing due to that.

error when starting the app:
![Screenshot from 2023-01-20 14-42-59](https://user-images.githubusercontent.com/90524466/213698078-a7b157cf-8d38-4ab7-8397-f0fc75af89fb.png)